### PR TITLE
Chore: Update GitHub PR template

### PR DESCRIPTION
**Pull request by silvenon**
**What is the purpose of this pull request? (put an "X" next to item)**

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
[ ] Add autofixing to a rule
[ ] Add a CLI option
[ ] Add something to the core
[x] Other, please explain: Updated this PR template

**What changes did you make? (Give an overview)**

1. Make them all sound the same, i.e. like a sentence
1. Added/suggested another item to the checklist: "Change the build process"

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -4,14 +4,15 @@
...
-[ ] Other, please explain:
+- [ ] Documentation update
+- [ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/bug-report.md))
+- [ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-proposal.md))
+- [ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/master/templates/rule-change-proposal.md))
+- [ ] Add autofixing to a rule
+- [ ] Add a CLI option
+- [ ] Add something to the core
+- [ ] Changes to build process
+- [ ] Other, please explain:
```
Comment (j-f1) [inline .github/PULL_REQUEST_TEMPLATE.md]: Could you change these to complete the sentence “This PR will…” (i.e. “Fix a bug“, “Change the build process”)?

**Comment by gyandeeps** [comment]
I dont think we want to make this change as after this change every PR will start showing tasks with only 1 complete out 7. Which has a highlight is very confusing and it will confuse more when somebody actually put a task list inside the PR.

**Comment by kaicataldo** [comment]
Thanks for the pull request! As noted, the current template is actually intentional and so I don't think we'll land this. We really appreciate you taking the time to contribute, regardless!

**Comment by silvenon** [comment]
Hey, no problem about rejecting the PR, but I didn't understand why. What is the intention behind the current template? Which commits of these three you don't want?

**Comment by mysticatea** [comment]
The checklist syntax causes confusion because it's shown in PR list. The intention of the current template is "choose one of them", not a task list.

image

**Comment by silvenon** [comment]
@mysticatea thanks for explaining, now I get it. 👍 I accidentally "corrected" it twice already. 😅

I removed that commit and updated the PR description. If you aren't interested in those other changes, feel free to close. 😉

**Comment by platinumazure** [comment]
I'm on the fence about including a "Change the build process" option (mostly because it feels rare enough that it could go under "Other"), but I appreciate the consistency in options' grammatical structure. :+1:

@eslint/eslint-team Any opinions on the addition of a "Change the build process" option to the PR template?

**Comment by kaicataldo** [comment]
@platinumazure I don't have a strong opinion either way. Happy to go with what everyone else think on this one.

**Comment by not-an-aardvark** [comment]
¯\\\_(ツ)_/¯ I'm also indifferent.

**Comment by silvenon** [comment]
Initially I submitted this PR thinking that the checklist part was a mistake. This PR doesn't add much value otherwise, so I'm closing it.

**Comment by kaicataldo** [comment]
@silvenon Thanks for taking the time to contribute!

**Pull request closed without merging**